src/ble/att/mod.rs

```diff
@@ -289,16 +302,16 @@ impl<A: Attributes> AttributeServer<A> {
         pdu: Pdu,
         responder: &mut L2CAPResponder,
     ) -> Result<(), AttError> {
-        match pdu.params {
+        let mut buf = [0u8; 16];
+        let mut writer = ByteWriter::new(&mut buf);
```

Comment:
Would be nice to directly use the `L2CAPResponder` instead, but we can refactor this later. The fixed-size buffer isn't very nice.

Reply:
I definitely agree